dvc/scm/git/__init__.py

```diff
@@ -287,9 +302,32 @@ def install(self):
                 "[ ! -d .git/rebase-merge ]",
             ],
             "checkout",
+            hook_path_fn,
+        )
+        self._install_hook("pre-commit", [], "status", hook_path_fn)
+        self._install_hook("pre-push", [], "push", hook_path_fn)
+
+        if use_pre_commit_tool:
+            self._integrate_pre_commit_tool()
+
+    def _integrate_pre_commit_tool(self):
+        if not which("pre-commit"):
+            raise DvcException("pre-commit is not installed")
+
+        conf = pre_commit_tool_conf(
+            self._pre_commit_tool_hook_path("pre-commit"),
+            self._pre_commit_tool_hook_path("push"),
+            self._pre_commit_tool_hook_path("post-checkout"),
         )
-        self._install_hook("pre-commit", [], "status")
-        self._install_hook("pre-push", [], "push")
+
+        conf_yaml = os.path.join(self.root_dir, ".pre-commit-config.yaml")
+        if not os.path.isfile(conf_yaml):
+            check_call(["pre-commit", "install"])
```

Comment:
This one is needed in case user has ` .pre-commit-hooks.yaml`, but no config? Or am I missing something?